padglobal/padglobal.py

```diff
@@ -938,6 +975,8 @@ def _get_emojis(self):
         if final_cmd != cmd:
             await message.channel.send(inline('Corrected to: {}'.format(final_cmd)))
         result = self.c_commands[final_cmd]
+        while result in self.c_commands:
```

Comment:
Can you guarantee this will never infinite loop?

Reply:
done

Reply:
How is this done? It's literally the same code.

Reply:
it's guaranteed by earlier code.  Unless someone manually edits the JSON file, it's safe

Reply:
OK, but comment that instead of 'done'. Done is generally used if you accept a proposed change.